packages/api-queries/src/plans.ts

```diff
@@ -1,13 +1,12 @@
 import { fetchPlans, fetchPlansDetails } from '@automattic/api-core';
 import { queryOptions } from '@tanstack/react-query';
 
-export const plansQuery = ( coupon: string, locale: string ) => {
+export const plansQuery = ( coupon: string = '' ) => {
 	const params = new URLSearchParams();
 	coupon && params.append( 'coupon_code', coupon );
-	params.append( 'locale', locale );
```

Comment:
We don't need locale as it will injected automatically [here](https://github.com/Automattic/wp-calypso/blob/9630b3e143d2cb9a9658f533408561751df0f38c/client/lib/wp/localization/index.js#L46)